app/src/main/java/org/listenbrainz/android/ui/screens/profile/BaseProfileScreen.kt

```diff
@@ -187,14 +187,20 @@ fun BaseProfileScreen(
                     if(currentTab.value == ProfileScreenTab.LISTENS){
                         when(isLoggedInUser) {
                             true -> AddListensButton()
-                            false -> when(uiState.listensTabUiState.isFollowing){
-                                true -> UnFollowButton(username = username, onUnFollowClick = {
-                                    onUnfollowClick(it)
-                                })
-                                false -> FollowButton(username = username, onFollowClick = {
-                                    onFollowClick(it)
-                                })
-                            }
+                            false->
+                                Box() {
```

Comment:
Please remove unnecessary `Box` wrapper.

Reply:
I see, this is a fault in the component itself, have a look if you can fix it ;)